src/components/Events/EventForm.vue

```diff
@@ -301,6 +301,9 @@ export default {
       if (this.event.spotsAvailable == null) {
         newSubmitErrors.spotsAvailable = 'required';
       }
+      if (this.event.price == null) {
```

Comment:
Wait actually, is it required on the backend? I more meant that we may want to null check before we make the comparison. Because we have a default, they shouldn't have to enter this? Or is there a default in the form too?